PreviewsSupport/remove_extensions.rb

```diff
@@ -0,0 +1,19 @@
+interfaces = Dir['PreviewsSupport.xcframework/**/*.swiftinterface']
```

Comment:
This removes the extensions from the built framework's swiftinterface files so they can be linked on a machine that doesn't have a modified SDK

Reply:
Maybe we can put these in a `PreviewsSupport/bin` dir?